amazon-ec2/2.3/modules/ROOT/pages/amazon-ec2-connector-examples.adoc

```diff
@@ -18,49 +28,45 @@ amazon.region=<Region>
 amazon.sessionToken=<Session Token>
 ----
 +
-. Drag the HTTP Listener component to the canvas, and specify the following values: +
-* *Host*: `localhost`
-* *Port*: `8081`
-. Drag the Amazon EC2 Connector *Create volume* operation next to the
-HTTP Listener component.
-. Configure the Amazon EC2 connector by adding a new global element.
-. Click the plus sign next to the *Connector Configuration* field.
-. Configure the global element:
-+
-[%header,cols="30s,70a"]
-|===
-|Studio Field |Value
-|Display Name |Listener
-|Extension Configuration a| If an HTTP Listener configuration doesn't exist:
-
-1. Click the plus sign (+) to add a new HTTP Listener configuration.
-2. Provide the following values: +
-* Host: `localhost`
-* Port: `8081`
-3. Click *OK*.
-|Path |/createVolume
-|===
-+
-. Drag the Amazon EC2 Connector *Create volume* operation next to the HTTP Listener component.
-. Configure the EC2 connector by adding a new Amazon EC2 global element and click *+* next to the *Connector Configuration* field.
+.. Save the `mule-artifact.properties` file in the project's `src/main/resources` directory.
+. Drag the *HTTP Listener* component to the canvas.
+. Add a global element for *HTTP Listener*:
+.. Select *Listener* on the canvas.
+.. Click the plus sign next to the *Connector Configuration* field.
+.. In the *Global Element Properties* dialog, provide the following values: +
+* *Name* +
+`Listener`
+* *Host* +
+`localhost`
+* *Port* +
+`8081`
++
+The following image shows an example of the *Listener* global element configuration:
++
+image::http-listener-config.png[Global element configuration for HTTP Listener]
++
+.. Click *OK*.
+. Drag the Amazon EC2 Connector *Create volume* operation next to the *Listener* component.
```

Comment:
Consider searching GLOBALLY for "listener" and verifying whether we are referring to a "global element" or a "component" and whether the UI label is only *Listener* or *HTTP Listener*, just to verify precision.

Reply:
It's just Listener. But generically, it's an HTTP listener. If you are giving instructions it is HTTP Listener because the operation belongs to HTTP connector.